src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.product-card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  color: $main-accent;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+  transition: $transition-duration;
+
+  &__image-block {
+    margin: auto;
+  }
+
+  &__image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-top: 16px;
+  }
+
+  &__reviews-count {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    margin: 0;
+    color: $secondary;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    margin-top: 16px;
+    height: 40px;
+    border: none;
+    border-radius: 5px;
+    background-color: $blue-accent;
+    color: $white;
+    text-transform: uppercase;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    transition: $transition-duration;
+
+    &:hover {
+      border: 1px solid $blue-accent;
+      background-color: $white;
+      color: $blue-accent;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The transform scale on hover might be too large and could affect the layout of surrounding elements. According to the checklist, ensure that interactive elements do not cause significant layout shifts. Consider using a smaller scale factor or another effect that doesn't impact layout.